cmd/bucket-replication.go

```diff
@@ -2675,3 +2688,85 @@ func (p *ReplicationPool) queueMRFHeal(file string) error {
 	}
 	return nil
 }
+
+// load replication stats from disk
+func (p *ReplicationPool) loadReplStats() (rs map[string]BucketReplicationStats, e error) {
+	data, err := readConfig(p.ctx, p.objLayer, replicationStatsPath)
+	if err != nil && err != errConfigNotFound {
+		return rs, err
+	}
+	if len(data) == 0 {
+		return rs, nil
+	}
+
+	if len(data) <= 4 {
+		return rs, fmt.Errorf("replication stats: no data")
+	}
```

Comment:
No need to return error here, relax it such that we can overwrite the new usage? @poornas